Assets/Scripts/Networking/NotWormsNetworkManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Characters;
+using Mirror;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Networking
+{
+    public class NotWormsNetworkManager : NetworkManager
+    {
+        [SerializeField]
+        private TurnController turnControllerPrefab;
+
+        public override void OnServerSceneChanged(string sceneName)
+        {
+            if (SceneManager.GetActiveScene().name.StartsWith("Network"))
```

Comment:
This is a bit of a hack I'm copying from a tutorial I'm doing. I think I'm a few videos away from figuring out a better way. I think it'll get fixed with a lobby system.